sdk/python/setup.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 The Kubeflow Authors
+# Copyright 2022 The Kubeflow Authors
```

Comment:
Adding the current year is optional, in case doing that, we should retain the original year, so it becomes 2018-2022.